src/types/types.ts

```diff
@@ -128,6 +130,39 @@ export type MatchesPosition<T> = Partial<
 export type Hit<T = Record<string, any>> = T & {
   _formatted?: Partial<T>
   _matchesPosition?: MatchesPosition<T>
+  _rankingScore?: number
+  _rankingScoreDetails?: Record<string, any>
+}
+
+export type RakingScoreDetails = {
```

Comment:
Is there any way to add comments to these types, to make them highlight the IDEs of people using it?

Reply:
Not that I'm aware of :/ I've searched and couldn't find anything either

Reply:
Yes, I realised I forgot to use `RankingScoreDetails` hehe. But to answer your question this is how IDe works w/ typescript types. 

<img width="735" alt="Screenshot 2023-07-11 at 11 53 29" src="https://github.com/meilisearch/meilisearch-js/assets/33010418/8c8b68b2-5f5a-499c-8eda-4eb2d62e3151">
<img width="510" alt="Screenshot 2023-07-11 at 11 53 55" src="https://github.com/meilisearch/meilisearch-js/assets/33010418/e89c62bc-a156-48a0-a116-c90f6e6d6769">

By clicking on `go to type definition` you'll be redirection to the `RankingScoreDetails` type.


Now, I can add a comment like this:

<img width="530" alt="Screenshot 2023-07-11 at 11 55 53" src="https://github.com/meilisearch/meilisearch-js/assets/33010418/09198d56-4785-4f96-b258-c6f77b995ee0">
<img width="446" alt="Screenshot 2023-07-11 at 11 57 05" src="https://github.com/meilisearch/meilisearch-js/assets/33010418/c7a26056-e3bb-4e88-bc10-4467243c025a">

Which would appear like that if I hover over the type (see last line of second screen):

<img width="448" alt="Screenshot 2023-07-11 at 11 57 00" src="https://github.com/meilisearch/meilisearch-js/assets/33010418/f67fec50-1271-4c21-9213-b671976e9636">